solaris/SolarisProcessList.c

```diff
@@ -71,12 +71,13 @@ ProcessList* ProcessList_new(UsersTable* usersTable, Hashtable* pidMatchList, ui
 static inline void SolarisProcessList_scanCPUTime(ProcessList* pl) {
    const SolarisProcessList* spl = (SolarisProcessList*) pl;
    int cpus = pl->cpuCount;
-   kstat_t* cpuinfo = NULL;
-   int kchain = 0;
+   kstat_t* cpu = NULL;
```

Comment:
Does this need distinct cpu and cpu_info or can it be folded into one?
(They look essentially the same in the stat_lookup calls below.)

Reply:
the same variable can certainly be used; the aim to use distinct ones was to have the variable name match the kstat module.

changed the variable name back to `cpuinfo` and using it for both kstat lookups.

Reply:
Great, thank you very much.
Merry Christmas, may the Sun always shine :-)